hail/Makefile

```diff
@@ -252,18 +252,18 @@ upload-artifacts: $(WHEEL)
 .PHONY: install-editable
 install-editable: $(PYTHON_VERSION_INFO) $(INIT_SCRIPTS)
 install-editable: python/README.md $(PYTHON_JAR)
-	-$(PIP) uninstall -y hail
+	-$(PIP) uninstall -y hail || true
```

Comment:
this `-` should suppress failure due to nonzero exit status, is it not?

Reply:
Oh I see. I didn't know about that. I saw:

```
Cannot uninstall requirement hail, not installed
Makefile:265: recipe for target 'install-on-cluster' failed
make: [install-on-cluster] Error 1 (ignored)
```

but I guess the `ignored` part is coming from that `-` and so it's all fine?

Reply:
It should be. This https://www.gnu.org/software/make/manual/make.html#Errors-in-Recipes describes the syntax. We shouldn't need `|| true`.